Gruntfile.js

```diff
@@ -3,194 +3,47 @@ module.exports = function (grunt) {
 
   var path = require('path');
 
-  grunt.loadNpmTasks('grunt-contrib-clean');
-  grunt.loadNpmTasks('grunt-contrib-coffee');
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-less');
-  grunt.loadNpmTasks('grunt-contrib-copy');
-  grunt.loadNpmTasks('grunt-contrib-qunit');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-ember-templates');
-  grunt.loadNpmTasks('grunt-jsdoc');
-  grunt.loadNpmTasks('grunt-neuter');
   grunt.loadNpmTasks('grunt-banner');
-  grunt.loadNpmTasks('grunt-text-replace');
-  grunt.loadNpmTasks('grunt-bower-task');
+  grunt.loadNpmTasks('grunt-broccoli');
+  grunt.loadNpmTasks('grunt-contrib-uglify');
   grunt.loadNpmTasks('grunt-release-it');
+  grunt.loadNpmTasks('grunt-text-replace');
+
+  // TODO(azirbel): We should register Ember Table, with its version, to Ember.Libraries
 
   // Project configuration.
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
     banner: '/*!\n* <%=pkg.name %> v<%=pkg.version%>\n' +
             '* Copyright 2012-<%=grunt.template.today("yyyy")%> Addepar Inc.\n' +
-            '* See LICENSE.\n*/',
+            '* See LICENSE.md.\n*/',
```

Comment:
Super nitpicky but I'd vote to remove the trailing `.`. This isn't a complete sentence anyways.